docs/UserGuide.md

```diff
@@ -54,6 +54,10 @@ command to run the application.<br>
 
 <box type="info" seamless>
 
+**Notes about the Screenshots:**<br>
+
+* Each Screenshot is taken based on the state of the data after running the command example stated. The original state of the data before running the command is that of the base Sample Data provided.
```

Comment:
maybe could add: based on the **original** state of data
~command example~ -> example command  
The original state of the data before running the command is that of the base Sample Data provided.